### ISSUE TITLE
UNinstallation instruction 

### ISSUE THREAD

**Issue by koroed06** [xvzc/SpoofDPI#189]
### Version

1.1(6)

### Operating System

Mac OS Sonoma 14.6.1

### How are you running SpoofDPI?

running by transfer Icon to Apps and start

### Description

after run SpoofDPI I have the same problems with playing YouTube and I have problems with every VPN connection - it doesn't work. Please say me and other step-by-step with images HOW TO UNINSTALL this utility??

**Comment by Ledorub** [comment]
Could you provide the output of `scutil --proxy`?

**Comment by koroed06** [comment]
> provide

scutil --proxy
<dictionary> {
  ExceptionsList : <array> {
    0 : *.local
    1 : 169.254/16
  }
  FTPPassive : 1
  HTTPEnable : 1
  HTTPPort : 8080
  HTTPProxy : 127.0.0.1
  HTTPSEnable : 1
  HTTPSPort : 8080
  HTTPSProxy : 127.0.0.1
}

**Comment by Ledorub** [comment]
Does your VPN work if you turn SpoofDPI off?

**Comment by toert** [comment]
Just try to reboot your Mac

**Comment by xvzc** [comment]
And your vpn not working with spoof-dpi is normal behavior because there's nothing listening to your http request on `127.0.0.1:8080` while you are on your vpn connection.

**Comment by koroed06** [comment]
> Does your VPN work if you turn SpoofDPI off?

no, it doesn't - it doesn't change country and ip-adress the same

**Comment by koroed06** [comment]
> I think your proxy setting's still enabled may be you can try turn on the spoof-dpi again and quit it normally

how to "quit it normally"?

**Comment by xvzc** [comment]
if you ran it on terminal then just press ctrl + c

**Comment by koroed06** [comment]
> And your vpn not working with spoof-dpi is normal behavior because there's nothing listening to your http request on `127.0.0.1:8080` while you are on your vpn connection.

what is the solution?
I want to delete SpoofDPI and have a basic network settings which I can surf internet with or without VPN-connections

**Comment by xvzc** [comment]
You should just disable your proxy settings. If you use Wi-fi, go to Wi-Fi -> Details -> Proxies and disable HTTP, HTTPS proxy

**Comment by Ledorub** [comment]
> I want to delete this utility and back settings basically

Try running SpoofDPI without any other VPNs and proxies, then stop it with Ctrl+C. What is the output of `scutil -proxy` after stopping the server?

**Comment by koroed06** [comment]
> > I want to delete this utility and back settings basically
> 
> Try running SpoofDPI without any other VPNs and proxies, then stop it with Ctrl+C. What is the output of `scutil --proxy` after stopping the server?

How does App run at Terminal? after that I can use "Ctrl+C" and "scutil --proxy"

**Comment by koroed06** [comment]
@Ledorub @xvzc now my Anydesk connections are loosed and I can't work correctly

how can I delete SpoofDPI clear and give back connection settings basically?

**Comment by Ledorub** [comment]
> How to run App at Terminal? after that I can use "Ctrl+C" and "scutil --proxy"

Sorry, I missed that comment. You can run the app with `spoofdpi` command.

Provide the output of:
1. `ps aux | grep spoofdpi`
2. `scutil --proxy`

**Comment by koroed06** [comment]
> > How to run App at Terminal? after that I can use "Ctrl+C" and "scutil --proxy"
> 
> Sorry, I missed that comment. You can run the app with `spoofdpi` command.
> 
> Provide the output of:
> 
> 1. `ps aux | grep spoofdpi`
> 2. `scutil --proxy`

What is "spoofdpi command"? Where can I read all Terminal command for SpoofDPI with using "use "Ctrl+C"" for exit

I use Terminal and step-by-step run command from your comment

_666!!! ~ % ps aux | grep spoofdpi
666  1427   0,0  0,0 410596064    800 s000  R+    4:32     0:00.00 grep spoofdpi
666!!! ~ % scutil --proxy
<dictionary> {
  ExceptionsList : <array> {
    0 : *.local
    1 : 169.254/16
  }
  FTPPassive : 1
  HTTPEnable : 1
  HTTPPort : 8080
  HTTPProxy : 127.0.0.1
  HTTPSEnable : 1
  HTTPSPort : 8080
  HTTPSProxy : 127.0.0.1
}_

**Comment by Ledorub** [comment]
Search for `proxy` in Settings and disable these options.
<img width="445" alt="Screenshot 2024-08-29 at 16 40 42" src="https://github.com/user-attachments/assets/9f5e1584-1963-44b1-8d6a-539daabbf5c8">

**Comment by koroed06** [comment]
> Search for `proxy` in Settings and disable these options. <img alt="Screenshot 2024-08-29 at 16 40 42" width="445" src="https://private-user-images.githubusercontent.com/19720324/362711569-9f5e1584-1963-44b1-8d6a-539daabbf5c8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjQ5NDAxNDgsIm5iZiI6MTcyNDkzOTg0OCwicGF0aCI6Ii8xOTcyMDMyNC8zNjI3MTE1NjktOWY1ZTE1ODQtMTk2My00NGIxLThkNmEtNTM5ZGFhYmJmNWM4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MjklMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODI5VDEzNTcyOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTE0ZDUyOThjOTYyY2U4YjJkMzlkMWM2ODNmNmE4NWQyMGRkYTg0NzAyMWQ0OGM3M2UyODdkYTkwOWJiYzhlM2UmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.-KEFtnrmC9opeUStVXWXCmLxRrSBRnoOBdKqaMEdOkI">

is it solution for all problems?

after disable options VPN-connection doesn't work correctly because in options App it doesn't change ip-adress

**HOW TO DELETE THIS APP AND GIVE BACK BASIC CONNECTION OPTIONS???**

**Comment by Ledorub** [comment]
> is it solution for all problems?

SpoofDPI does not change other options.

> VPN-connection doesn't work correctly

Which VPN connection?

> HOW TO DELETE THIS APP AND GIVE BACK BASIC CONNECTION OPTIONS???

Which method did you use to install the app?

**Comment by xvzc** [comment]
Go to `System Settings` -> `Network` -> `Details` on your current network -> `Proxies`
and disable proxy settings for HTTP and HTTPS. That way you can disable all the settings that spoofdpi has set for you. If this doesn't work for you, we can't give you any help.

And please tell me how you've installed spoofdpi then we can let you know how to delete it, but most of the time leaving it in your system will not cause any side effect.

**Comment by koroed06** [comment]
@Ledorub 
> > is it solution for all problems?
> 
> SpoofDPI does not change other options.
> 
> > VPN-connection doesn't work correctly
> 
> Which VPN connection?
I use Hide.Me VPN-connection
> 
> > HOW TO DELETE THIS APP AND GIVE BACK BASIC CONNECTION OPTIONS???
> 
> Which method did you use to install the app?

I download App from Github and drag to Apps folder after that run it

@xvzc 
> Go to `System Settings` -> `Network` -> `Details` on your current network -> `Proxies` and disable proxy settings for HTTP and HTTPS. That way you can disable all the settings that spoofdpi has set for you. If this doesn't work for you, we can't give you any help.
> 
> And please tell me how you've installed spoofdpi then we can let you know how to delete it, but most of the time leaving it in your system will not cause any side effect.

after disable proxy options anything doesn't work correctly (YouTube, VPN, AnyDesk)
I download App from Github and drag to Apps folder after that run it

**Comment by xvzc** [comment]
Is your network totally not working?

**Comment by xvzc** [comment]
move the app to the trash. that's how you delete an app on MacOS

**Comment by koroed06** [comment]
> Is your network totally not working?

yes, it's

> move the app to the trash. that's how you delete an app on MacOS

the problems continue to surface

**Comment by DopDrop** [comment]
Вы ебанные мрази

**Comment by DopDrop** [comment]
Сука. Вы ебаные пидарасы

**Comment by DopDrop** [comment]
Ебаные вы пидары
